docs/user_guide/io/mrsim_IO.rst

```diff
@@ -131,12 +152,13 @@ A list of methods in a :ref:`simulator_api` object can be serialized to a file.
 custom DAS method and serialize it to a file using the method
 :meth:`~mrsimulator.Simulator.export_methods`.
 
-.. code-block:: python
+.. plot::
+    :context: close-figs
 
     from mrsimulator import Simulator
     from mrsimulator.methods import Method2D
-    from mrsimulator.method.events import SpectralEvent
-    from mrsimulator.method.spectral_dimensions import SpectralDimension
+    from mrsimulator.method.event import SpectralEvent
+    from mrsimulator.method.spectral_dimension import SpectralDimension
```

Comment:
I think we should bring the Events and SpectralDimension to the method level. Instead of
```
from mrsimulator.method.event import SpectralEvent
from mrsimulator.method.spectral_dimension import SpectralDimension
```
it will be
```from mrsimulator.method import SpectralDimension, SpectralEvent, MixingEvent```

Reply:
I think this is a good idea. This should probably get done before v0.7 is released since we are adding ``MixingEvent``.

Will begin working on a new PR soon